portfolio/src/main/webapp/style.css

```diff
@@ -11,35 +11,38 @@ title {
 #greeting-container {
   margin-top: 20px;
 }
+
+h1 {
+    text-align: center;
+}
+
 p1 {
```

Comment:
So related to the comment above regarding not using p1, p2, p3 because these tags do not exist, I would just give the tags a class instead to change their styling.